pallets/pallet-bonded-coins/src/traits.rs

```diff
@@ -0,0 +1,9 @@
+use sp_runtime::DispatchError;
+
+pub trait FreezeAccounts<AccountId, AssetId> {
+	type Error: Into<DispatchError>;
+
+	fn freeze(caller: &AccountId, who: &AccountId, asset_id: &AssetId) -> Result<(), Self::Error>;
+
+	fn thaw(caller: &AccountId, who: &AccountId, asset_id: &AssetId) -> Result<(), Self::Error>;
```

Comment:
another small thing: in the fungibles traits, the asset id always comes first - so how about
```suggestion
	fn thaw(asset_id: &AssetId, who: &AccountId) -> Result<(), Self::Error>;
```

Reply:
[2bb01ba](https://github.com/KILTprotocol/kilt-node/pull/770/commits/2bb01baea1178fb023c214a71e18473fa9eff2c3)